spec/features/merchants/invoices/show_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "rails_helper"
+
+RSpec.describe "Merchant Invoice Show Page", type: :feature do
+  before :each do
```

Comment:
This may cause issues since we're requiring rails helper. Might be best to extracting it and just calling the defined name.